src/components/collection/Summary.js

```diff
@@ -240,6 +244,27 @@ function Summary({
             )}
           </Typography>
           <Authorship itemId={itemId} author={creator} isLoading={isLoading} />
+          {createdAt && (
+            <div id={SUMMARY_CREATED_AT_CONTAINER_ID}>
+              <Typography variant="h6" gutterBottom>
+                Created At
+              </Typography>
+              <Typography variant="p" gutterBottom>
+                {new Date(createdAt).toLocaleDateString()}
```

Comment:
I would use maybe a nice library to have a nicer display, like `date-fns` or any recommendation from @dialexo

Reply:
Very good point, i would suggest luxon, the immutable successor to moment: https://moment.github.io/luxon/